packages/elements-core/src/components/MosaicTableOfContents/TableOfContents.tsx

```diff
@@ -331,53 +343,56 @@ const Node = React.memo<{
   meta?: React.ReactNode;
   showAsActive?: boolean;
   isInResponsiveMode?: boolean;
+  makeSlugAbsoluteRoute?: boolean;
   onClick?: (e: React.MouseEvent, forceOpen?: boolean) => void;
   onLinkClick?(): void;
-}>(({ item, depth, meta, showAsActive, isInResponsiveMode, onClick, onLinkClick = () => {} }) => {
-  const activeId = React.useContext(ActiveIdContext);
-  const isActive = activeId === item.slug || activeId === item.id;
-  const LinkComponent = React.useContext(LinkContext);
+}>(
+  ({ item, depth, meta, showAsActive, isInResponsiveMode, makeSlugAbsoluteRoute, onClick, onLinkClick = () => {} }) => {
+    const activeId = React.useContext(ActiveIdContext);
+    const isActive = activeId === item.slug || activeId === item.id;
+    const LinkComponent = React.useContext(LinkContext);
 
-  const handleClick = (e: React.MouseEvent) => {
-    if (isActive) {
-      // Don't trigger link click when we're active
-      e.stopPropagation();
-      e.preventDefault();
-    } else {
-      onLinkClick();
-    }
+    const handleClick = (e: React.MouseEvent) => {
+      if (isActive) {
+        // Don't trigger link click when we're active
+        e.stopPropagation();
+        e.preventDefault();
+      } else {
+        onLinkClick();
+      }
 
-    // Force open when clicking inactive group
-    if (onClick) {
-      onClick(e, isActive ? undefined : true);
-    }
-  };
+      // Force open when clicking inactive group
+      if (onClick) {
+        onClick(e, isActive ? undefined : true);
+      }
+    };
 
-  return (
-    <Box
-      as={LinkComponent}
-      to={item.slug}
-      display="block"
-      textDecoration="no-underline"
-      className="ElementsTableOfContentsItem"
-    >
-      <Item
-        id={getHtmlIdFromItemId(item.slug || item.id)}
-        isActive={isActive || showAsActive}
-        depth={depth}
-        title={item.title}
-        icon={
-          NODE_TYPE_TITLE_ICON[item.type] && (
-            <Box as={Icon} color={NODE_TYPE_ICON_COLOR[item.type]} icon={NODE_TYPE_TITLE_ICON[item.type]} />
-          )
-        }
-        meta={meta}
-        isInResponsiveMode={isInResponsiveMode}
-        onClick={handleClick}
-      />
-    </Box>
-  );
-});
+    return (
+      <Box
+        as={LinkComponent}
+        to={makeSlugAbsoluteRoute && !/^\//.test(item.slug) ? `/${item.slug}` : item.slug}
```

Comment:
It took me a minute to find this change, so I'm just marking it for other reviewers.

Is there an advantage to using a regular expression here instead of `!item.slug.startsWith('/')`?

Reply:
old habits die hard